libmobilecoin/include/slip10.h

```diff
@@ -9,35 +9,25 @@
 extern "C" {
 #endif
 
-/* ==== Types ==== */
-
-typedef struct _McSlip10Indices McSlip10Indices;
-
-/* ==== McSlip10Indices ==== */
-
-McSlip10Indices* MC_NULLABLE mc_slip10_indices_create(void);
-
-void mc_slip10_indices_free(
-  McSlip10Indices* MC_NULLABLE indices
-);
-
-bool mc_slip10_indices_add(
-  McSlip10Indices* MC_NONNULL indices,
-  uint32_t index
-)
-MC_ATTRIBUTE_NONNULL(1);
-
 /* ==== McSlip10 ==== */
 
 /// # Preconditions
 ///
-/// * `out_key` - length must be >= 32.
-bool mc_slip10_derive_ed25519_private_key(
-  const McBuffer* MC_NONNULL seed,
-  const McSlip10Indices* MC_NONNULL path,
-  McMutableBuffer* MC_NONNULL out_key
+/// * `mnemonic` - must be a nul-terminated C string containing valid UTF-8.
```

Comment:
```suggestion
/// * `mnemonic` - must be a null-terminated C string containing valid UTF-8.
```

Reply:
I've been using `nul` everywhere. I think if we're going to change it to `null`, we should do it all at once.